VladimirPopov_WebForms/templates/webforms/form/default.phtml

```diff
@@ -58,271 +79,294 @@ if ($this->getForm()->getIsActive() == \VladimirPopov\WebForms\Model\Form::STATU
+                    </fieldset>
+                <?php endforeach; // foreach 'fields_to_fieldsets' ?>
+
+                <?= $this->getLayout()->createBlock(
+                    'VladimirPopov\WebForms\Block\Script\Gdpr',
+                    null,
+                    ['data' => [
+                        'uid' => $_uid
+                    ]]
+                )->setForm($this->getForm())->toHtml();
+                ?>
 
+                <div
+                    id="submit<?= $_uid ?>"
+                    class="actions-toolbar"
+                    <?php if ($this->getSubmitVisibility() == 'hidden'): ?>
+                        style="display: none"
+                    <?php endif; ?>
+                >
+                    <?php if ($this->getForm()->useCaptcha()): ?>
+                        <?php $_captcha = $this->getCaptcha() ?>
+                        <div class="recaptcha<?= $block->escapeHtmlAttr($_captcha->getVersion()) ?>">
+                            <?= $_captcha->getHTML() ?>
+                        </div>
+                    <?php endif; ?>
+
+                    <?php if ($this->honeypot()): ?>
                         <?= $this->getLayout()->createBlock(
-                            'VladimirPopov\WebForms\Block\Script\Gdpr',
+                            'Magento\Framework\View\Element\Template',
                             null,
-                            ['data' => [
-                                'uid' => $_uid
-                            ]]
-                        )->setForm($this->getForm())->toHtml();
-                        ?>
-                        <div
-                            id="submit<?= $block->escapeHtmlAttr($_uid) ?>"
+                            [
+                                'data' => [
+                                    'webform_id' => $this->getForm()->getId(),
+                                    'template' => 'VladimirPopov_WebForms::webforms/scripts/honeypot.phtml'
+                                ]
+                            ]
+                        )->toHtml(); ?>
+                    <?php endif; ?>
+                    <div class="primary">
+                        <button
+                            type="submit"
                             class="
-                                <?php if ($this->getSubmitVisibility() == 'hidden'): ?>
-                                    display-none
-                                <?php endif; ?>
+                                button
+                                margin-top-m
+                                action
+                                submit
+                                primary
                             "
+                            id="webform_<?= $_uid ?>_submit_button"
+                            title="<?= $block->escapeHtmlAttr(__($this->getForm()->getSubmitButtonText())) ?>"
+                            name="field[submit]"
+                            onclick="return false;"
```

Comment:
is i required here? there is a `preventDefault` in script, it doesn't work?

Reply:
Looks like it can be removed